src/pages/groupComparison/ClinicalData.tsx

```diff
@@ -937,6 +937,7 @@ export default class ClinicalData extends React.Component<
                                     ? this.highlightedRow.qValue
                                     : null
                             }
+                            sortOption={'sortByAlphabet'}
```

Comment:
we can probably delete this now that we made `sortOption` optional and we've taken care of the case when `sortOption` is not `sortByCount`